features/step_definitions/common_steps.rb

```diff
@@ -244,7 +244,7 @@
 end
 
 Then /^I see a (success|error|notice) flash message containing #{MAYBE_VAR}$/ do |type, message|
-  flash_message = page.find(:css, ".dm-alert.dm-alert--#{type}", wait: false)
+  flash_message = page.find(:css, ".dm-alert.dm-alert--#{type}, div[class^='banner-']", wait: false)
```

Comment:
Nice! If we have this, do we still need the `@skip-staging` tags?

Reply:
I don't _think_ so? I swiped the tags from #769 by @lfdebrux but then he suggested this change.

Reply:
I'm also unsure about this and exactly the order of how it fits in the workflow. If we clarify this it would be good to document it!

Reply:
In theory this change should mean that we don't need the tags anymore.

However when I tested this by running this branch against staging with `(. config/staging.sh ; bundle exec cucumber --strict --tags @skip-staging)` I still got failures due to the flash messages 🤔

Reply:
`div.flash-message-container` works better for the second selector (that's on me, typed something out without testing it before)

Reply:
Done.

Is that `config/staging.sh` something you've created locally and does it have a specific configuration?

Reply:
I copied `config/common.sh` and updated the environment variables based on values in `digitalmarketplace-credentials/vars/staging.yaml`.

Reply:
OK, this is working locally for me. Since @lfdebrux mentioned issues when removing `@skip-staging` I'm keeping those tags in and I think this should be good to merge?

Reply:
The tests now pass against staging, so we can remove these tags and also the existing ones in #769.

Reply:
Done!